tests/python/Lut1DTransformTest.py

```diff
@@ -16,50 +16,38 @@
     np = None
 
 import PyOpenColorIO as OCIO
+from TransformsBaseTest import TransformsBaseTest
 
 
-class Lut1DTransformTest(unittest.TestCase):
+class Lut1DTransformTest(unittest.TestCase, TransformsBaseTest):
```

Comment:
Apologies for missing the previous review of `TransformBaseTest`. Have you considered making that base class inherit from `unittest.TestCase`, and then have the individual transform test cases inherit from only that, instead of using the base class as a mixin? I'm curious if the `tearDown` method from the base class gets called. It might, but would be worth confirming.

Reply:
Hey Michael! So the reason for this is explained here: https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1288#discussion_r572638482

I would love to know if there is a better way if you could show me. But the result of my testing is that `TransformBaseTest` base class would get call on if it inherits from `unittest.TestCase` on initialization. So without being able to initialize the `self.tr` to the individual transforms object, it is `None` which causes multiple call to `test_direction` to fail. Let me know what you think about this and thank you.

Reply:
@michdolan @meimchu Is the discussion now completed?

Reply:
Sorry for the late reply. Thanks for sharing the link to that thread @meimchu . Sounds like you've already explored that option, so this approach should be fine. Thanks again for all your work on this!